worker/src/RTC/SctpAssociation.cpp

```diff
@@ -449,7 +449,7 @@ namespace RTC
 		// via onSendSctpData.
 		this->listener->OnSctpAssociationBufferedAmount(this, this->sctpBufferedAmount);
 
-		const int ret = usrsctp_sendv(
+		const ssize_t ret = usrsctp_sendv(
```

Comment:
Why this? The function literally returns int which maybe it could be any negative value while ssize_t can only be -1 and 0 and positive integers.